### ISSUE TITLE
Feedback from some interviews

### ISSUE THREAD

**Issue by ppannuto** [c4cs/c4cs.github.io#270]
Workload:
  - "1 credit class baited me into taking it in a semester where I shouldn't have taken it"
    -- For people who have a busy semesters how do we avoid pushing people over the edge
    -- Related: More aggressive reporting of grades / progress to stop people from falling behind
 - It's not hard in terms of difficulty, it's hard in terms of time & dilligence
    -- "The HW is the kinda stuff I want to put off, it's like calc homework or filling out applications"
    -- "dilligence" recurring theme: feels like the weekly workload / floating assignments become overwhelming, hard to keep track of what needs to be done / doing things every week
 - Remind people that it's a Flex Tech, and that 1 credit of Flex Tech != 1 credit of Gen Elective; helps to justify the weekly work
    -- Actually if this is Flex Tech, it's a really solid balance and the right amount of work

Taking the class concurrent with 280, I would highly recommend that for anyone.

HW/Lec balance
  - Lecture covers about half the HW material, then go explore on your own
    -- That's pretty much the goal (whoo!)

HW
  - Somehow still need to re-emphasize the using git in a class project
    -- Maybe week4/5 or s/t, should have a "what class project are you using git with" question

Advanced / Grading
  - 10 + 2 + 2.. maybe 2 is a little too dramatically low. Maybe allow for more points for the first extra that's not tied to sections?
  - The 10 is really intimidating, 10 = 10% of your grade [which is the point..], but it's scary and if it's missed it feels insurmountable
    -- [I think this can be helped by the more aggressive grade feedback]
  - Maybe the other way is to have a cap on the number of EC points in a section as opposed to a universal point pool

Adv2:
  - Too unclear what the correct answer is / what we're looking for
  - If this stays at this point in the semester, worth greater emphasis that it's safe to try a few things, mess up a few things, DL a fresh copy of the repo, etc

Adv6:
  - The gradescope submission idea needs to go away

**Comment by mterwill** [comment]
> 10 + 2 + 2.. maybe 2 is a little too dramatically low. Maybe allow for more points for the first extra that's not tied to sections?

Agreed. I've talked to a couple of people who are in that boat. I know we thought it was a good idea to split everything up, but it doesn't seem like it's made much of a difference -- there were 57 OH queue requests served last Friday [last day of the section], so it seems to me we basically just pushed the problem around. I'm in favor of offering a forgiveness week like last semester.

---
> The gradescope submission idea needs to go away

Funny; I'd heard some positive feedback from people whose schedules/situations made it difficult to get to OH.  Though I guess ahw7 (an open source contribution) had the same effect.

7's one of my favorites. Definitely think it's a valuable hurdle to cross during a class -- now students can say "oh, I've done that before," instead of "GitHub what?"

**Comment by the-alex** [comment]
> The gradescope submission idea needs to go away

I tend to agree with the aggression towards the online submission of advanced homework. It doesn't really accomplish what we want for the assignments. It really just seems to make our office hours a bit more sparse. I still had students coming in, asking if their answers were correct. I could have just checked them off there.

**Comment by ppannuto** [comment]
Had a chat with one more student today. One new thing that came up was trying to find mechanisms to make clear that OH aren't just advanced submission time and to try to better encourage people who are working on the HW to swing by and ask for help.

I don't have a great mechanism for this -- ideas at the moment stuff like, offer a floating 1pt of HW EC that you can claim once by visiting OH for HW help at least 48 hours before it's due, but that's a little too.. something.

**Comment by mterwill** [comment]
Notes from our staff meeting today:

1. Dedicate one OH to quasi-discussion
2. Change advanced homework grading scheme to be a little more forgiving
3. Advanced homework for week 1 -- maybe vagrant, docker, etc.
